### ISSUE TITLE
torhid folder is never created anywhere

### ISSUE THREAD

**Issue by GuiltyDescent** [ThrillQuks/Pitraix#3]
Line 483 `hostnamef, err := readFile(filepath.Join(path, name, name + "hid", "hostname"))` requests the hostname/onion address from a file that never gets to exist in the subdirectories of the main path.

I spent an hour or two trying to figure out ways around or to fix this but I could not.

This is the terminal error that blocks me from going further
`hostname read error: open /home/user/Documents/Pitraix/tor/torhid/hostname: no such file or directory`

With this, the program just loops thanks to line 487 and retries setting up tor once more (and fails).

I set up a tor hidden service separately on the previous update but ran into a completely different issue that I forgot about and decided to wait for this new patch that the README file noted was in the works.

This feels like such a great project! Thank you for your contribution.

**Comment by GuiltyDescent** [comment]
I would also like to note that there are double folders? I don't know if that's intended so I just kept the lines that make it so.

**Comment by GuiltyDescent** [comment]
* **What operating system are you using? (exact version)**
Client will be tested on both kali-linux-2022.3-amd64 virtualbox image from kali.org (Kernal: Linux 5.18.0-kali5-amd64) and Microsoft Windows 10 Home (10.0.19044) VM
I will disable VM detection for those tests
* **What operating system is the host running? (exact version)**
GNU/Linux POP!_OS 22.04 (Kernal Version: 5.19.0-76051900-generic 64-bit)
* **Are you using Release or master?**
I am using the latest release of v1.0 (I did use the master before this first release though)
* **Did you or the host have TOR previously installed?**
The host (POP!_OS) has the TOR browser installed from a flatpak
* **Are you running both OPER and lyst payload on same machine?**
Payload and OPER are run on separate systems (VM & bare metal)

I will attempt to host on a fresh kali install on a VM and report the results, I made the mistake of forgetting to do so on this version.
